new/client-rb-logger.md

```diff
@@ -0,0 +1,39 @@
+---
+RFC: unassigned
+Title: Logging for chient.rb
+Author: Zeal Jagannatha <zeal@fb.com>
+Status: Draft
+Type: Standards Track
+Tracking:
+  - https://github.com/chef/chef/issues/5526
+---
+
+# Logging for client.rb
+
+Output from client.rb doesn't get logged to the log file specified with --logfile.
+If you want to save the output from client.rb along-side the chef logs, you need to redirect stdout, rather than using chef's built-in logging.
+I'd like to make Chef::Log accessible from the client.rb so messages logged during client.rb execution can be stored using chef's built-in logging.
+
+## Motivation
+
+    As a chef user,
+    I want to log output from client.rb into the same log file as the chef output,
+    so that client.rb output can be tracked in the same method as the chef run.
+
+## Specification
+
+1. Create a [Logger](https://github.com/chef/mixlib-log) instance before client.rb is initialized.
+1. Log messages on this logger will be written to stdout, and saved.
```

Comment:
What's the behavior if you log something in the config file and then later the config file raises an exception? I could totally buy either (major) approach (drop messages or try to get them to the real logger) but we should pick one :)